packages/g/google-libphonenumber.json

```
@@ -0,0 +1,41 @@
+{
+  "description": "The up-to-date and reliable Google's libphonenumber package for node.js for parsing, formatting, and validating international phone numbers.",
+  "filename": "dist/libphonenumber.js",
+  "homepage": "https://github.com/ruimarinho/google-libphonenumber",
+  "keywords": [
+    "e164",
+    "format",
+    "formatting",
+    "international",
+    "libphonenumber",
+    "number",
+    "phone",
+    "phonenumber",
+    "rfc3966",
+    "standardize"
+  ],
+  "license": "MIT",
+  "name": "google-libphonenumber",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ruimarinho/google-libphonenumber.git"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "google-libphonenumber",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

```suggestion
          "*.js"
```
We try to avoid globstars when possible, especially open ones -- it looks like we just need to match the JS files here so this should be sufficient.